<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addForm</class>
 <widget class="QWidget" name="addForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить экспонат</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Фонд музея</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Section">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSubselection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Подраздел фонда музея</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Subselection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonFond_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Основной фонд</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonFond_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Научно-вспомогательный фонд</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Номер учётной записи</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
